app/controllers/teams/users_controller.rb

```diff
@@ -0,0 +1,37 @@
+class Teams::UsersController < AuthenticatedController
+   before_action -> { authorize team, policy_class: TeamUsersPolicy }
+
+    def remove_confirm
+      @team = team
```

Comment:
I suspect we don't actually need to call team here, since we should be calling it before every action, so the instance variable is already set. I vote we remove this line if everything still works without it.